--- /usr/bin/neofetch	2020-08-02 22:31:59.000000000 +0200
+++ /usr/bin/neofetch.sh	2020-09-24 15:33:09.480962026 +0200
@@ -5841,6 +5841,35 @@
  .`                                 `/
 EOF
         ;;
+	 "Hipster"*)
+            set_colors 7 0	
+            read -rd '' ascii_data <<'EOF'
+${c2}                ..     .....               
+             .:::::::::;,'  ',             
+            .:'.;;;;;::::,',:::.           
+            ,:'   ...,::::::::::           
+            ;:::;:::::::::::::::;          
+    .,;::::  .',;::::::::::::::::'         
+   .:::::::'       ...;::::::::::, ..      
+    .;:::::::;'.       ;:::::::::';:::;.   
+      .,::::::::::;'....;:::::::::::::::,  
+         .';::::::::::::::::::::::::::::'  
+         ;   .,;:::::::::::::::::::::::.   
+        ${c1}.MK  ${c2}''; ..',;:::::::::::::;'.     
+    ${c1}.:o: .Kc WMMK;       .......           
+ kMMMMMMXl;. ,NMMMWOxxk0XKk0X0kK,          
+ ;MMMMMMMMM.   XMMMMMMMxdKOXMMMX           
+  0MMMMMMMK    'MMMNOxd:';cNMMW,  .coxxd' 
+  .NMMMMMMc      .       '  od..cxNMMMMN. 
+   .NMMMMW           .kXOXx.  xMMMMMMMN.  
+    .KMMMX.               '. :MMMMMMMK.   
+      lWMMX.                 0MMMMMWl     
+       .dWMW:               ;MMMMWd.      
+          cKMNo.           xMMMKc         
+            .:xKk:'.    .:XKx:.           
+                 ;d0NN0d;
+EOF
+        ;;
 
         "artix_small"*)
             set_colors 6 6 7 1
